en/components/grids_templates/sorting.md

```diff
@@ -117,9 +117,27 @@ public ngOnInit() {
 
 <div class="divider--half"></div>
 
-@@if (igxName === 'IgxGrid') {
+@@if (igxName === 'IgxGrid' || igxName === 'IgxTreeGrid') {
 #### Remote Sorting
-You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#ondatapreload) and [`onSortingDone`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#onsortingdone) outputs. More information on how to use it you can find in the `@@igComponent Virtualization and Performance` [documentation](virtualization.md#remote-sortingfiltering-virtualization).
+You can provide @@igComponent's remote sorting by subscribing to [`onDataPreLoad`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#ondatapreload) and either the [`sortingExpressionsChange`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#sortingexpressionschange) or the [`onSortingDone`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#onsortingdone) outputs.
+@@if (igxName === 'IgxGrid') {More information on how to use them you can find in the `@@igComponent Virtualization and Performance` [documentation](virtualization.md#remote-sortingfiltering-virtualization).}
+
+> [!NOTE]
+> The [`sortingExpressionsChange`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#sortingexpressionschange) output is fired whenever a change to the sorting expressions has occurred. This output is triggered prior to the actual sorting operation, unlike the [`onSortingDone`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#onsortingdone) output, which is triggered after the sorting has finished.
+
+When remote sorting is provided, usually we do not need the built-in sorting of the @@igComponent. We can disable it by setting the [`sortStrategy`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#sortstrategy) input of the @@igComponent to the `NoopSortingStrategy` instance. When this strategy is used, the sorting functionality is left entirely for the developer to implement.
+
+> [!NOTE]
+> The [`sortStrategy`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#sortstrategy) of the **@@igComponent** is of different type compared to the [`sortStrategy`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#sortstrategy) of the **column**, since they work in different scopes and expose different parameters.
+
+```html
+<@@igSelector [allowFiltering]="true" [sortStrategy]="noopSortStrategy">
+</@@igSelector>
+```
+
+```typescript
+public noopSortStrategy = NoopSortingStrategy.instance();
+```
```

Comment:
There is no need to duplicate the information about remote sorting implementation here. It is enough just to have a link to the Virtualization and Performance topic. You could move the sortStrategy note to the **Sorting through the API** section

Reply:
Addressed.